src/types.ts

```diff
@@ -138,6 +138,11 @@ export type Feedback = {
    * feedback.
    */
   promptId?: FeedbackPrompt["promptId"];
+
+  /**
+   * Source, depending on how the user was asked
+   */
+  source?: "api" | "manual" | "prompt" | "sdk" | "widget";
```

Comment:
Should we remove `api` and `manual` from this enum? The SDK does not use them and probably better not to be advertised.

Reply:
Yep, good shout